.azure-pipelines/templates/test-all.yml

```
@@ -27,3 +27,4 @@ jobs:
       # Avoid max step limits
       ${{ if eq(check.checkName, 'datadog_checks_base') }}:
         validate: true
+        validate_codeowners: true
```

Should the value instead be false here?

```suggestion
        validate_codeowners: false
```

And should a `validate_codeowners: ${{ parameters.validate_codeowners}}` be added to the parameter list above?

---

Good catch, updated.